AzureData/Source/DocumentClient.swift

```diff
@@ -226,7 +226,12 @@ public class DocumentClient {
 
 
     // replace
-    // TODO: replace
+    public func replace (inCollectionWithId collectionId: String, fromDatabase databaseId: String, policy: DocumentCollection.IndexingPolicy, callback: @escaping (Response<DocumentCollection>) -> ()) {
```

Comment:
Please update function signature to the following: 
`public func replace (collectionWithId collectionId: String, inDatabase databaseId: String, usingPolicy policy: DocumentCollection.IndexingPolicy, callback: @escaping (Response<DocumentCollection>) -> ())`

Reply:
I initially thought of this implementation. But it seems less flexible, in case in the future, new parameters that do not match the properties and/or the structure of `DocumentCollection` could be passed to the request body. 
Your comment makes sense and I'll revert but I just want to get your thoughts @colbylwilliams.